### PR TITLE
fix bug, when dim of aten::size.int(Tensor self, int dim) -> (int) is…

### LINKED ISSUE #396: 🐛 [Bug] aten::size.int(Tensor self, int dim) -> (int) when dim<0,  the result is wrong
##  Bug Description

aten::size.int(Tensor self, int dim) -> (int) when dim<0,  the result is wrong

## To Reproduce

Add a unit test in test_aten_evaluators.cpp
```
TEST(Evaluators, SizeConvertsCorrectly) {
  const auto graph = R"IR(
      graph(%0 : Tensor):
        %1 : int = prim::Constant[value=-1]()
        %2 : int = prim::Constant[value=-2]()
        %3 : int = aten::size(%0, %1)
        %4 : int = aten::size(%0, %2)
        %5 : int[] = prim::ListConstruct(%3, %4)
        %6 : Tensor = aten::view(%0, %5)
        return (%6))IR";

  auto g = std::make_shared<torch::jit::Graph>();
  torch::jit::parseIR(graph, &*g);

  auto in = at::randint(1, 10, {3, 3}, {at::kCUDA});

  auto params = trtorch::core::conversion::get_named_params(g->inputs(), {});
  auto jit_results = trtorch::tests::util::RunGraph(g, params, {in});

  params = trtorch::core::conversion::get_named_params(g->inputs(), {});
  auto trt_results = trtorch::tests::util::RunGraphEngine(g, params, {in});

  ASSERT_TRUE(trtorch::tests::util::almostEqual(jit_results[0], trt_results[0].reshape_as(jit_results[0]), 2e-6));
}
```
result is that :
image

## Environment

> Build information about the TRTorch compiler can be found by turning on debug messages

 - PyTorch Version (e.g., 1.0):
 - CPU Architecture:
 - OS (e.g., Linux):
 - How you installed PyTorch (`conda`, `pip`, `libtorch`, source):
 - Build command you used (if compiling from source):
 - Are you using local sources or building from archives:
 - Python version:
 - CUDA version:
 - GPU models and configuration:
 - Any other relevant information:

### PR DESCRIPTION
**Pull request by guoruoqian**
… negative

Signed-off-by: Ruoqian Guo <ruoqiang@nvidia.com>

# Description

Fix bug, when dim of aten::size.int(Tensor self, int dim) -> (int) is negative.

Fixes #396 

## Type of change

Please delete options that are not relevant and/or add your own.

- Bug fix (non-breaking change which fixes an issue)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist:

- [x] My code follows the style guidelines of this project (You can use the linters)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas and hacks
- [ ] I have made corresponding changes to the documentation
- [x] I have added tests to verify my fix or my feature
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by narendasan** [comment]
Do you know why the entire file changed?

**Comment by guoruoqian** [comment]
> Do you know why the entire file changed?

**Comment by guoruoqian** [comment]
> Do you know why the entire file changed?

I run this command "bazel run //tools/linter:cpplint -- //..."  to ensure that my code matches the style guidelines. Then git add the file "core/conversion/evaluators/aten.cpp".  I think the linters changed the entire file. I just modified aten::size part to fix bug like this:
image

Do I need to change other parts of the code back?

**Comment by guoruoqian** [comment]
> Do you know why the entire file changed?

Now eveything in `core/conversion/evaluators/aten.cpp` is back.

**Comment by narendasan** [comment]
Is this really a breaking change?

File: core/conversion/evaluators/aten.cpp [hunk 1]
```diff
@@ -180,9 +180,16 @@ auto aten_registrations TRTORCH_UNUSED =
                         auto dim = args.at(n->input(1)).unwrapToInt();
                         if (tensor_var.isITensor()) {
                           auto tensor = tensor_var.ITensor();
-                          return util::toVec(tensor->getDimensions())[dim];
+                          auto dims = util::toVec(tensor->getDimensions());
+                          auto nbDims = tensor->getDimensions().nbDims;
+                          if (dim < 0)
```
Comment (narendasan) [inline core/conversion/evaluators/aten.cpp:185]: Style thing: use the brackets even if its a one liner

File: core/conversion/evaluators/aten.cpp [hunk 2]
```diff
@@ -180,9 +180,16 @@ auto aten_registrations TRTORCH_UNUSED =
...
+                            dim += nbDims;
+                          return dims[dim];
                         } else {
                           auto tensor = tensor_var.unwrapToTensor();
+                          auto nbDims = tensor.sizes().size();
+                          if (dim < 0)
```
Comment (narendasan) [inline core/conversion/evaluators/aten.cpp:191]: Same here

**Review by narendasan — COMMENTED** [review]
style things, otherwise lgtm

**Comment by narendasan** [comment]
Can you resolve conflicts here? @guoruoqian

**Comment by guoruoqian** [comment]
> Can you resolve conflicts here? @guoruoqian

Yeah，i have resolved the conflicts.

**Pull request merged**